src/universal/components/AddTeamMemberModal.tsx

```diff
@@ -122,6 +125,13 @@ class AddTeamMemberModal extends Component<Props, State> {
   render () {
     const {closePortal, error, submitting} = this.props
     const {invitees, successfulInvitations, rawInvitees} = this.state
+    if (isDemoRoute()) {
```

Comment:
the demo modal doesn't require any of the state or props that this component uses, so it should probably be promoted.
I'd make it a lazy component on AddTeamMemberAvatarButton, that way we only load it for the demo

Reply:
Thanks for the tip, I’ll look at this.